### PR TITLE
log the TLS Transport Parameters

### PR DESCRIPTION
**Pull request by marten-seemann**

### CODE REVIEW

File: internal/handshake/transport_parameters.go [hunk 1]
```diff
@@ -174,3 +174,9 @@ func (p *TransportParameters) getTransportParameters() []transportParameter {
 	}
 	return params
 }
+
+// String returns a string representation, intended for logging.
+// It should only used for IETF QUIC.
+func (p *TransportParameters) String() string {
+	return fmt.Sprintf("&handshake.TransportParameters{StreamFlowControlWindow: %#x, ConnectionFlowControlWindow: %#x, MaxBidiStreamID: %d, MaxUniStreamID: %d, OmitConnectionID: %t, IdleTimeout: %s}", p.StreamFlowControlWindow, p.ConnectionFlowControlWindow, p.MaxBidiStreamID, p.MaxUniStreamID, p.OmitConnectionID, p.IdleTimeout)
```
Comment (lucas-clemente) [inline internal/handshake/transport_parameters.go]: Why not %#v?

Comment (marten-seemann) [reply internal/handshake/transport_parameters.go]: Because it displays durations as floats.

Comment (lucas-clemente) [reply internal/handshake/transport_parameters.go]: Is that worth the risk of not remembering to add new fields to the log statement?

Comment (marten-seemann) [reply internal/handshake/transport_parameters.go]: I'd say so. I don't expect (m)any additions to the transport parameters anyway.

**Comment by marten-seemann** [comment]
@lucas-clemente: LGTY?

**Pull request merged**